### ISSUE TITLE
wappalyzer detects 'Plentymarkets' and apps.json has 'plentymarkets'

### ISSUE THREAD

**Issue by AdityaAnand1** [AliasIO/wappalyzer#3289]
Notice the difference in upper and lower cases :/

These should be the exact same.

**Comment by AliasIO** [comment]
The regular expressions are case insensitive so that's expected. Is that not right?

**Comment by AdityaAnand1** [comment]
It would be much easier to have both the detected names as well as the source-of-truth, i.e, apps.json have precisely the same technology names everywhere. 

If it's done this way, then a simple equality check would do in our applications, SQL queries etc. Even though `lowerCase()`-ing and regex-ing would work, equality checks are always better (and more performant when used in SQL queries searching among millions of rows!)

**Comment by AliasIO** [comment]
Ah, I see what you mean. `apps.json` is the source of truth, where are you seeing the upper-case version?

It looks like the casing was corrected recently:

https://github.com/AliasIO/wappalyzer/pull/3149/files#diff-557d7d60c04092a680ce152f367618c0L8496

**Comment by AdityaAnand1** [comment]
Yes, exactly. 

So, I've been storing responses as is in my database. I encountered this when I updated apps.json, wappalyzer scraper. After updating, some of the rows showed "plentymarkets" while others "Plentymarkets" - which, since the name is used as primary key, it started showing up as two different things.

So, ideally, the names should be treated as immutable from the beginning, i.e, once set they should never be renamed. This would be the ideal scenario, where we take greater care to ensure that when some new tech is added to wappalyzer, we're 100% sure the name is accurate, so future fixes are avoided/minimized. When the underlying tech name itself changes, then we don't have much choice, but we can still try to keep names consistent everywhere.
